docs/_tabsets/install.qmd

```diff
@@ -14,6 +14,7 @@ backends = [
     {"name": "DataFusion", "module": "datafusion"},
     {"name": "Druid", "module": "druid"},
     {"name": "DuckDB", "module": "duckdb"},
+    {"name": "Exasol", "module": "exasol"},
```

Comment:
this probably shouldn't be in this PR?

Reply:
Whoops, not sure how that got there. I did a rebase merging main to get the change needed for the last query maybe I messed something up. I will close this PR and open a new one because I'm not sure what happened. 

I'll include the edits suggested below too.